NEWS.md

```diff
@@ -2,13 +2,27 @@
 
 This file contains a summary of user-visible or important changes, in
 the style of the GNU coding standards.  By user-visible, we include
-changes relevant for those building unison from source.
+changes relevant for those building unison from source.  We omit
+most bugfixes and minor improvements.
 
 As of 2022, this file (NEWS.md at top level) is used for news.
 
 Some software has a "changelog" file that records all changes, but
 unison uses git history for that, and thus there is no changelog file.
 
+## Changes in 2.53.0
+
+NOT YET RELEASED
+
+  * Change OCaml requirement to >= 4.08
+  * Clean up a variety of unmaintained and unused bits, mainly
+    build-related
+  * Change GUI to use lablgtk3 instead of lablgtk2
```

Comment:
Perhaps only mention GTK. lablgtk is a library name that few users know.

Reply:
Good point - I adjusted the wording and force-pushed, but I think it's important to leave lablgtk* present because that is a build dependency, and NEWS is written for an audience including those that build from source.